src/main/java/io/aiven/kafka/connect/opensearch/OpensearchSinkConnectorConfig.java

```diff
@@ -173,7 +173,7 @@ private static void addConnectorConfigs(final ConfigDef configDef) {
         configDef.define(
                 CONNECTION_URL_CONFIG,
                 Type.LIST,
-                ConfigDef.NO_DEFAULT_VALUE,
```

Comment:
@frochse I beleive setting the default URL to arbitrary choosen value is not a good configuration practice (specifically to workaround the UI limitations).

Reply:
Oke i understand the point of view. So in this case the UI should resolve this and a default value for the URL is not allowed? or is there a construct which tackles this in a acceptable way?

Reply:
> Oke i understand the point of view. So in this case the UI should resolve this and a default value for the URL is not allowed? 

Correct

>  or is there a construct which tackles this in a acceptable way?

I am not entirely sure how you configure the connector from UI, but if you need `CONNECTION_URL_CONFIG` to be set, you could always supply the configuration with some preset value (fe `https://eshost1:9200`), would it work in your case?

Reply:
Yes it would and we are doing that. The thing is that UI's who deal with connect just read the config and if a default is not there and it is mandatory they don't like it. Other mandatory fields are supplied with a default in the current code. So indeed i thought it would be nice if a preset value like esbhost1:9200 would be present in this config.  I understand the opinion that ui's should be able to handle this null value on mandatory fields, but in practise, lets say it depends. 

So it would help the usability of the opensearch connector to allow this small change. Although you could say it is not the best of practises.

Reply:
> The thing is that UI's who deal with connect just read the config and if a default is not there and it is mandatory they don't like it.

This is the semantics of a required configuration: You must specify a value, and if there is not one provided, Kafka Connect will report a validation error. The UIs cannot resolve this error, and notify the user that the configuration they provided is incorrect.

All Kafka Connect UIs must handle validation failures, whether the value is specified or not. If this is causing extra poor behavior in the UI, then the UI must be adjusted to accommodate validation errors for unspecified configurations. Many other connectors use the same semantics for required configurations, not just this connector and this configuration.